awx/api/permissions.py

```diff
@@ -120,15 +120,17 @@ def check_permissions(self, request, view, obj=None):
         return result
 
     def has_permission(self, request, view, obj=None):
-        logger.debug('has_permission(user=%s method=%s data=%r, %s, %r)', request.user, request.method, request.data, view.__class__.__name__, obj)
-        try:
-            response = self.check_permissions(request, view, obj)
-        except Exception as e:
-            logger.debug('has_permission raised %r', e, exc_info=True)
-            raise
-        else:
-            logger.debug('has_permission returned %r', response)
-            return response
+        response = self.check_permissions(request, view, obj)
+        logger.debug(
+            'has_permission(user=%s method=%s, %s, %r) returned %r',
+            request.user,
+            request.method,
+            view.__class__.__name__,
+            obj,
+            response,
+            extra={'volume_tag': 'rbac'},
+        )
+        return response
```

Comment:
These are logs that are disabled by the LOGGING setting, and I'm bringing them in from the cold - letting people turn it on with the `LOG_AGGREGATOR_LOGGERS` setting here.